Description: Replace inclusion of unixodbc_conf.h with preprocessor #defines
 unixodbc_conf.h is an arch-specific header that contains #defines indicating
 how unixODBC was built. In general, programs should check for and define
 SIZEOF_LONG_INT and HAVE_LONG_LONG at compile-time.
 .
 Some programs do not #define SIZEOF_LONG_INT, resulting in the inclusion of
 unixodbc_conf.h. This header #defines many macros, including sizeof(long),
 and indicates the existence of 'long long' and various system headers.
 .
 Since we aren't packaging unixodbc_conf.h, we insert the relevant unixODBC
 preprocessor #defines here. These are identical on all Debian arches.
 .
 Upstream is working on an alternative solution.
Author: Hugh McMaster <hugh.mcmaster@outlook.com>
Last-Update: 2022-02-22
Bug: https://github.com/lurcher/unixODBC/issues/99
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1005887
Forwarded: not-needed

--- a/include/sqltypes.h
+++ b/include/sqltypes.h
@@ -49,7 +49,11 @@
  */
 
 #ifndef SIZEOF_LONG_INT
-#include "unixodbc_conf.h"
+#define HAVE_LONG_LONG 1
+#define HAVE_PWD_H 1
+#define HAVE_SYS_TYPES_H 1
+#define HAVE_UNISTD_H 1
+#define SIZEOF_LONG_INT __SIZEOF_LONG__
 #endif
 
 #ifndef SIZEOF_LONG_INT
